### PR TITLE
Update team.md

### PR DESCRIPTION
**Pull request by vmnet8**
Dear teammates,
I finished step 1-9,  added myself in team.md. 
Hope to join the team soon.

### CODE REVIEW

**Comment by dogi** [comment]
@vmnet8 missing raw githack link ;)

**Review by dogi — CHANGES_REQUESTED** [review]
on hold until interview

**Comment by dogi** [comment]
also congrats @vmnet8

**Comment by vmnet8** [comment]
again? haha.
https://raw.githack.com/vmnet8-OLE/vmnet8-OLE.github.io/team/#!pages/vi/team.md
please check.

**Comment by vmnet8** [comment]
> also congrats @vmnet8

thank you :)

**Review by Liew211 — APPROVED** [review]

**Review by lmmrssa — APPROVED** [review]

File: pages/vi/team.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 |[Lillyxxx](profiles/lillyxxx.md)|2019-07-17|
 |[CalebProvost](profiles/CalebProvost.md)|2019-09-04|
 |[AnPham](profiles/phamduchongan93.md)|2019-09-07|
+|[vmnet8](profiles/vmnet8.md)|2019-09-11|
```
Comment (dogi) [inline pages/vi/team.md:19]: congrats @vmnet8 :)

**Review by dogi — APPROVED** [review]

**Review by CalebProvost — APPROVED** [review]

**Pull request merged**